tests/ote_cli/test_ote_cli_tools_anomaly_classification.py

```diff
@@ -171,9 +171,24 @@ def test_ote_deploy_openvino(template):
     assert run(['python3', '-m', 'pip', 'install', 'wheel'],
                cwd=os.path.join(deployment_dir, 'python'),
                env=collect_env_vars(os.path.join(deployment_dir, 'python'))).returncode == 0
-    assert run(['python3', '-m', 'pip', 'install', 'demo_package-0.0-py3-none-any.whl'],
+    
+    assert check_ote_sdk_commit_hash_in_requirements(os.path.join(deployment_dir, 'python', 'requirements.txt'))
```

Comment:
Is it possible to reduce code duplication here somehow?

Reply:
Sure, will do if this pr potentially can be merged

Reply:
refactored